src/page/options.ts

```diff
@@ -0,0 +1,22 @@
+import { exportStates, importStates } from "../TitleState";
+
+const inputImport = document.getElementById("import") as HTMLInputElement;
+inputImport.addEventListener("change", (fileEvent) => {
+  const file = (fileEvent.target as HTMLInputElement).files![0];
+  const reader = new FileReader();
+  reader.onload = async (readEvent) => {
+    const count = await importStates(readEvent.target!.result as string);
+    alert(`${count} 개 작품 불러오기 성공`);
+  };
+  reader.readAsText(file);
```

Comment:
혹시 [`file.text()`](https://developer.mozilla.org/en-US/docs/Web/API/Blob/text)도 괜찮을까 싶습니다. 도입된 지 어느 정도 되어서 버전 걱정 없이 쓸 수 있을 듯합니다.

```suggestion
  const file = (fileEvent.target as HTMLInputElement).files![0];
  const count = await importStates(await file.text());
  alert(`${count} 개 작품 불러오기 성공`);
```